python/cudf/cudf/io/parquet.py

```diff
@@ -666,7 +739,15 @@ def to_parquet(
             row_group_size_rows=row_group_size_rows,
             max_page_size_bytes=max_page_size_bytes,
             max_page_size_rows=max_page_size_rows,
-            **kwargs,
+            partitions_info=list(
+                zip(
+                    partition_offsets,
+                    np.roll(partition_offsets, -1) - partition_offsets,
```

Comment:
Try:
```python
list(zip(partition_offsets, np.diff(partition_offsets)))
```

Example:
```python
import numpy as np
offsets = [0, 10, 15, 25, 37]
print(list(zip(offsets, np.roll(offsets, -1) - offsets))[:-1])
print(list(zip(offsets, np.diff(offsets))))
```
Both lines print:
```
[(0, 10), (10, 5), (15, 10), (25, 12)]
```

Reply:
Or:
```python
[(i, j - i) for i, j in zip(offsets, offsets[1:])]
```
If you don't want to invoke numpy

Reply:
Works 👍